package.json

```diff
@@ -30,31 +30,31 @@
     "@babel/preset-env": "^7.21.5",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.21.5",
-    "@careevolution/mydatahelps-js": "^3.23.0",
```

Comment:
Removed this because it's in `peerDependencies`.

Reply:
I was always wondering if I was missing some reason why it was duplicated here.  😄